mobile/src/components/setting/setting.tsx

```diff
@@ -42,21 +48,51 @@ import { styles } from './styles';
 const Settings: FC = () => {
   const dispatch = useAppDispatch();
   const navigation = useAppNavigate();
-  const { userDetails, dataStatus } = useAppSelector(({ userDetails }) => ({
-    userDetails: userDetails.userDetails,
-    dataStatus: userDetails.dataStatus,
-  }));
+  const [selectedImage, setSelectedImage] = useState<Asset | null>();
+
+  const { user, userDataStatus, userDetails, userDetailsDataStatus } =
+    useAppSelector(({ auth, userDetails }) => ({
+      user: auth.user,
+      userDataStatus: auth.dataStatus,
+      userDetails: userDetails.userDetails,
+      userDetailsDataStatus: userDetails.dataStatus,
+    }));
 
   const maxDate = subtractYears(new Date(), UserAge.MIN);
   const minDate = subtractYears(new Date(), UserAge.MAX);
+  const selectionLimit = 1;
```

Comment:
Move outide component to <components>/common/constants/constants and use [quality criteria](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md) for constants(upper case)